examples/camera-app/linux/src/clusters/webrtc_provider/webrtc-provider-manager.cpp

```diff
@@ -226,7 +226,12 @@ void WebRTCProviderManager::ScheduleAnswerSend()
         CASESessionManager * caseSessionMgr = Server::GetInstance().GetCASESessionManager();
         VerifyOrDie(caseSessionMgr != nullptr);
 
-        caseSessionMgr->FindOrEstablishSession(mPeerId, &mOnConnectedCallback, &mOnConnectionFailureCallback);
+        // WebRTC Answer requires a large payload session establishment.
+        caseSessionMgr->FindOrEstablishSession(mPeerId, &mOnConnectedCallback, &mOnConnectionFailureCallback,
+#if CHIP_DEVICE_CONFIG_ENABLE_AUTOMATIC_CASE_RETRIES
+                                               1, nullptr,
+#endif // CHIP_DEVICE_CONFIG_ENABLE_AUTOMATIC_CASE_RETRIES
+                                               TransportPayloadCapability::kLargePayload);
```

Comment:
This suggests that we should have a version of FindOrEstablishSession that does the "default behavior" for the retry bits when CHIP_DEVICE_CONFIG_ENABLE_AUTOMATIC_CASE_RETRIES is defined but lets you control the transport payload capabilities and lets consumers who don't care not need to have this ifdef business at callsites....

Reply:
Added an overload of FindOrEstablishSession that omits those params and uses the defaults internally.